sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/PipelineJsonSamples.cs

```diff
@@ -3815,7 +3815,9 @@ public class PipelineJsonSamples : JsonSampleCollection<PipelineJsonSamples>
                     {
                         type: ""SapOpenHubSource"",
                         excludeLastRequest: false,
-                        baseRequestId: ""123""
+                        baseRequestId: ""123"",
+                        customRfcReadTableFunctionModule: ""fakecustomRfcReadTableFunctionModule"",
+                        sapDataColumnDelimiter: ""fakesapDataColumnDelimiter""
```

Comment:
this should only be one character. could you please change it to be "|". Thanks. similar for the SapTable one.